frontend/src/scenes/session-recordings/player/playerMetaLogic.tsx

```diff
@@ -258,7 +264,7 @@ export const playerMetaLogic = kea<playerMetaLogicType>([
     })),
     listeners(({ actions, values, props }) => ({
         loadRecordingMetaSuccess: () => {
-            if (values.sessionPlayerMetaData && !values.recordingPropertiesLoading) {
+            if (values.sessionPlayerMetaData) {
```

Comment:
it's safe to just always try and load this, since the target logic short circuits if already loaded